### PR TITLE
Adding How to sections

### PR DESCRIPTION
**Pull request by muralibasani**
- Adding structure for Concepts & How to
- Updating How to -> Installation pages
- Klaw Connectivity to Kafka clusters

### CODE REVIEW

File: docs/docs/howto/installation/configure-klaw-wizard.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+4. Test Kafka connectivity
+
+5. For existing Kafka clusters, migrate Topics and Acls to Klaw.
+
+6. Create Teams
+
+7. Create Users
+
+8. Create any other Roles and Permissions (Optional)
```
Comment (jlprat) [inline docs/docs/howto/installation/configure-klaw-wizard.rst:24]: The way this is structured right now it feels like it's either too few or too much. Let me explain, in here only the steps of the wizard are described, but using the "number points list" feels like each point should contain more information, maybe what can be tuned, or what are the preferred configs.

If the previous one is not the intended, I think we could change the format of this list to something less vertically spacious (so it doesn't feel like you are going to describe each of them).

File: docs/docs/howto/installation/run-docker.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+``http://<dockerhost>:9097/``
+
+5. Default Credentials to access Klaw
+
+    ::
+
+    KLAW_SUPERADMIN_DEFAULT_USERNAME=superadmin
+    KLAW_SUPERADMIN_DEFAULT_PASSWORD=kwsuperadmin123$$
+
+.. note::  Please change the above default credentials in your application
```
Comment (jlprat) [inline docs/docs/howto/installation/run-docker.rst:70]: I would put this higher up, and reversed. Something like this


.. note:: Please change the the default credentials in your application before running it

Here you can find the settings with their default value:

    ::

    KLAW_SUPERADMIN_DEFAULT_USERNAME=superadmin
    KLAW_SUPERADMIN_DEFAULT_PASSWORD=kwsuperadmin123$$

**Review by jlprat — CHANGES_REQUESTED** [review]
Thanks!
I left my feedback on the files themselves. I think the default password should be encouraged higher up and the wizard description feels strange.

File: docs/docs/howto/installation/configure-klaw-wizard.rst (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/docs/howto/installation/configure-klaw-wizard.rst:24]: > The way this is structured right now it feels like it's either too few or too much. Let me explain, in here only the steps of the wizard are described, but using the "number points list" feels like each point should contain more information, maybe what can be tuned, or what are the preferred configs.
> 
> If the previous one is not the intended, I think we could change the format of this list to something less vertically spacious (so it doesn't feel like you are going to describe each of them).

I could probably help with refining the content to make it sound better for the end user.

Comment (muralibasani) [reply docs/docs/howto/installation/configure-klaw-wizard.rst:24]: thanks @harshini-rangaswamy

File: docs/docs/howto/installation/configure-klaw-wizard.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+You can configure min max partitions, replication factors, prefix or suffix for topic names.
+
+3. Test Kafka connectivity
+
+After configuring environments, you may test the connectivity by clicking the validate button.
+
+
+.. image:: /../../../_static/images/EnvStatus.png
+
+3. Configure Tenant and Environment hierarchy
```
Comment (jlprat) [inline docs/docs/howto/installation/configure-klaw-wizard.rst:42]: This should be 4.

File: docs/docs/howto/installation/configure-klaw-wizard.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    dev1.kafkassl.truststore.pwd=trustme
+    dev1.kafkassl.truststore.type=JKS
+
+2. Configure Environments
+
+Environments are wrappers over your kafka clusters. You can have multiple environments connected to one Kafka cluster with different settings
+
+You can add any number of environments. Only users with SUPERADMIN roles (or check allowed permission) can add by default.
+
+You can configure min max partitions, replication factors, prefix or suffix for topic names.
```
Comment (jlprat) [inline docs/docs/howto/installation/configure-klaw-wizard.rst:33]: "min max partitions" doesn't sound like a sentence. Maybe "minimum or maximum number of partitions"?

**Review by jlprat — CHANGES_REQUESTED** [review]
There is no point 4, but twice point 3.

File: docs/docs/howto/installation/configure-klaw-wizard.rst (hunk 4 above)
Comment (muralibasani) [reply docs/docs/howto/installation/configure-klaw-wizard.rst]: Updated.

File: docs/docs/howto/installation/configure-klaw-wizard.rst (hunk 3 above)
Comment (muralibasani) [reply docs/docs/howto/installation/configure-klaw-wizard.rst]: Updated.

File: docs/docs/howto/installation/run-docker.rst (hunk 2 above)
Comment (muralibasani) [reply docs/docs/howto/installation/run-docker.rst]: Updated.

File: docs/docs/howto/installation/configure-klaw-wizard.rst (hunk 1 above)
Comment (muralibasani) [reply docs/docs/howto/installation/configure-klaw-wizard.rst]: Agree, each of them are elaborated now.

File: docs/docs/howto/installation/run-source.rst [hunk 5]
```diff
@@ -1,34 +1,32 @@
 Run Klaw from source
 ====================
 
-Klaw is a Java project, you will need `Maven <https://maven.apache.org/>`_ installed to run this project from source.
+Klaw is a Java project, and you need `Maven <https://maven.apache.org/>`_ installed to run this project from the source.
```
Comment (jlprat) [inline docs/docs/howto/installation/run-source.rst:4]: This is not the case anymore, right? As we have `mvnw`, this will download maven for you

**Review by jlprat — CHANGES_REQUESTED** [review]
Small pick about requiring maven installed. as we now have the wrapper, I think it's not needed

File: docs/docs/howto/installation/run-source.rst (hunk 5 above)
Comment (muralibasani) [reply docs/docs/howto/installation/run-source.rst]: Agree, updated.

**Review by jlprat — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**